tests/spark-it/src/test/scala/org/apache/celeborn/tests/spark/PushDataTimeoutTest.scala

```diff
@@ -34,30 +36,62 @@ class PushDataTimeoutTest extends AnyFunSuite
     logInfo("test initialized , setup celeborn mini cluster")
     val workerConf = Map(
       CelebornConf.TEST_CLIENT_PUSH_MASTER_DATA_TIMEOUT.key -> "true",
-      CelebornConf.TEST_WORKER_PUSH_SLAVE_DATA_TIMEOUT.key -> "true",
-      "celeborn.push.timeoutCheck.interval" -> "1s")
```

Comment:
no documentation or usage information could be found for the `celeborn.push.timeoutCheck.interval` parameter.